src/main/java/bio/terra/cli/businessobject/resource/GcsBucket.java

```diff
@@ -30,6 +30,7 @@ public class GcsBucket extends Resource {
   /** Deserialize an instance of the disk format to the internal object. */
   public GcsBucket(PDGcsBucket configFromDisk) {
     super(configFromDisk);
+    this.resourceType = Type.GCS_BUCKET;
```

Comment:
is this change related?

Reply:
This was added to the description, not the title since the title was already long earlier. Adding it now.